test/writable-keys-arrays-and-index-signatures.ts

```diff
@@ -0,0 +1,22 @@
+import { AssertTrue as Assert, IsExact } from "conditional-type-checks";
+import { WritableKeys } from "../lib";
+
+function testWritableKeys() {
+  type cases = [
```

Comment:
To avoid regression, please add the same list of test cases with primitives (without `Assert` and expected values)